### PR TITLE
☁️ get GenomeAnalysisTK from google cloud

### PR DESCRIPTION
**Pull request by dmiller15**
Closes https://github.com/d3b-center/bixu-tracker/issues/847

Ended up having to install the google-cloud-sdk to get this file as the https file in the ticket had protections and could not be downloaded with curl or wget. 

Test with the new docker: https://cavatica.sbgenomics.com/u/d3b-bixu/kf-germline-harmonization-dev/tasks/62b584be-54ad-4280-aa2b-b55dce5d1b95/

### CODE REVIEW

File: gatk/4.beta.1-3.5/Dockerfile [hunk 1]
```diff
@@ -4,10 +4,11 @@ MAINTAINER Yuankun Zhu (zhuy@email.chop.edu)
 ENV GATK4_VERSION 4.beta.1
```
Comment (migbro) [inline gatk/4.beta.1-3.5/Dockerfile:4]: It won't let me comment high, but here: `MAINTAINER Yuankun Zhu (zhuy@email.chop.edu)` might be good to switch to `LABEL` format while you're already here, to bring this back up to spec.Do you think doing a quick test run of `tools/gatk_haplotypecaller.cwl` would be good to ensure this works, or is that overkill?

**Review by migbro — CHANGES_REQUESTED** [review]
Biggest request is to just bring the maintainer label to spec...not sure if there is a standard docker "updated by" that could be added

File: gatk/4.beta.1-3.5/Dockerfile (hunk 1 above)
Comment (dmiller15) [reply gatk/4.beta.1-3.5/Dockerfile:4]: Updated.

**Review by migbro — APPROVED** [review]
Looks good, thanks!

**Review by sickler-alex — APPROVED** [review]

**Pull request merged**